fs/fs.go

```diff
@@ -137,10 +149,48 @@ func NewFsInfo(context Context) (FsInfo, error) {
 	fsInfo.addDockerImagesLabel(context, mounts)
 	fsInfo.addCrioImagesLabel(context, mounts)
 
-	glog.V(1).Infof("Filesystem UUIDs: %+v", fsInfo.fsUUIDToDeviceName)
-	glog.V(1).Infof("Filesystem partitions: %+v", fsInfo.partitions)
 	fsInfo.addSystemRootLabel(mounts)
-	return fsInfo, nil
+}
+
+func CheckFsInfoUpdateForever(fsInfo *RealFsInfo, context Context, changed chan bool) {
+	for {
+		if pollMountEvent(60000) {
```

Comment:
I would prefer using inotify over polling.

Reply:
At first I used inotify to watch `/proc/self/mountinfo` but when I attach an mount into my machine and `mount` to a directory, Nothing was watched, even though the file is quitely changed. 
Same result with watching `/proc/mounts` and `/proc/self/mounts` by inotify

After that I found two topics in SOF: 
https://stackoverflow.com/questions/1113176/how-could-i-detect-when-a-directory-is-mounted-with-inotify/24671215#24671215
https://stackoverflow.com/questions/5070801/monitoring-mount-point-changes-via-proc-mounts/5072579#5072579   , and tried it with golang.  
Or do you know some better ways to do capture mount/umount event?

Reply:
Try using fsnotify. https://github.com/fsnotify/fsnotify

Reply:
Still doesn't work. 
I init a FSWatcher by:
````
        files:="/proc/self/mountinfo"
        files1 := "/proc/mounts"
        files3 := "/proc/self/mounts"
        watcher , _ := newFSWatcher(files, files1, files3, file2)
````
and capture all Events and Errors:
````
for {
        select {
                case event := <-watcher.Events:
                        fmt.Println(event.Name,  event.Op)
                case err := <-watcher.Errors:
                        fmt.Println("inotify: %s", err)
        }
}
````
Still print nothing when i `mount /dev/vda1 /test` or `umount /test`